app/controllers/api/graphql_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module API
+  class GraphqlController < ActionController::API
+    # If accessing from outside this domain, nullify the session
+    # This allows for outside API access while preventing CSRF attacks,
+    # but you'll have to authenticate your user separately
+    # protect_from_forgery with: :null_session
+
+    def execute
+      render json: result
+    rescue StandardError => e
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/46106ccd13ad8d3333b75e32a92764a807848798/app/controllers/api/graphql_controller.rb#L12"></span>Has the variable name ‘e’</td>
    </tr>
  </tbody>
</table>